include/can/core/messages.hpp

```diff
@@ -536,8 +536,8 @@ using GetMotionConstraintsRequest =
 struct GetMotionConstraintsResponse
     : BaseMessage<MessageId::get_motion_constraints_response> {
     uint32_t message_index;
-    um_per_tick min_velocity;
-    um_per_tick max_velocity;
+    mm_per_tick min_velocity;
```

Comment:
Are we changing the resolution to mm_per_tick or um_per_tick?

Reply:
Yeah so we've been mistakenly using `um_per_tick` for velocity for a long time (not in calculation, just for naming the units), so this clears that/

So `mm_per_tick` for velocity and `um_per_tick_sq` for acceleration